app/api/ada/hardwareWallet/newTransaction.js

```diff
@@ -122,9 +127,9 @@ export async function newTrezorTransaction(
   }
 }
 
-function _derivePath(change: number, index: number): string {
+export function _derivePath(change: number, index: number): string {
   // Assumes this is only for Cardano and Web Yoroi (only one account).
-  return `${Config.trezor.DEFAULT_CARDANO_PATH}/${change}/${index}`;
+  return `${Config.wallets.CARDANO_FIRST_BIP_PATH}/${change}/${index}`;
```

Comment:
@sarveshwar-singh FIRST_BIP_PATH doesn’t sound great. We should user the standardized naming https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki